operator/config/rbac/role.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
```

Comment:
Would it not need get as well? Or get is available by default?

Reply:
We are not reading events just creating them, so think its ok